mantidproject/mantid
# Centre on SANSLoad

**Description of work.**
In SANS v2 GUI, workspaces are now centred on SANSLoad, before being moved.

Note that in SANSLoad the component property is not used, and the component is hard coded as "LAB". As outlined in #24529, currently both components are moved regardless of component.

This extra issue will be tackled in a separate PR, which will change SANSLoad back to using the component as provided as input.

**Report to:** nobody

**To test:**
1. Interfaces -> SANS -> SANS v2 GUI
2. Load a matching user and batch file from the attached data
3. Click the load button
4. In the ADS, expand the group workspace, and on each workspace right click and show instrument. 
5. In SANS -> Settings -> Mask click DisplayMask. 
Compare all of these images. 
Some workspaces from the ADS will show grey instruments, but the coloured ones should roughly match-up with the Displayed Mask. The most important part is that the centre of Display Mask and the Instruments matches

[PRdata.zip](https://github.com/mantidproject/mantid/files/2756090/PRdata.zip)


Fixes #23960 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
